src/guides/v2.3/javascript-dev-guide/javascript/requirejs.md

```diff
@@ -62,6 +62,28 @@ paths: {
 }
 ```
 
+Consider    the example of overwriting an HTML file in the adminhtml.
+In this example, the `max-length` value of the text-box in the `adminhtml` is altered. The HTML file is located at `vendor/magento/module_ui/view/base/web/templates/form/element/input.html`.
+
+1. Create a `requirejs-config.js` file under `app/code/<Vendor_Name>/<Module_Name>/view/base/` and add the following code:
+
+    ```javascript
+    var config = {
+        paths: {
+            'ui/template/form/element/input': '<vendor_name>_<module_name>/template/form/element/input'
+        }
+    };
+    ```
+
+1. Create an `input.html` file under `app/code/<Vendor_Name>/<Module_Name>/view/base/web/template/form/` and copy the contents of the `input.html` file from the `module_ui` template file.
+1. Change the maxlength value to `512`, which was originally set to `256`.
+1. Upgrade the Magento application using `bin/magento setup:upgrade`.
+1. Generate the dependency injection configuration using `bin/magento setup:di:compile`.
```

Comment:
Shell commands must be formatted using code fences (```) before and after the command (indented three spaces after a list item). The first set of code fences must be followed by the language (bash).

```bash
bin/magento setup:di:compile
```